GoMain/bin/res/configuration.toml

```diff
@@ -0,0 +1,78 @@
+[Writable]
```

Comment:
Is this a sample for the datastorage configuration?
Why don't you put the sample files in the [samples/datastorage folder](https://github.com/lf-edge/edge-home-orchestration-go/tree/master/samples)?

Reply:
+1 as @t25kim pointed out.

Reply:
Good point, incorporated